### DISCUSSION TITLE
2025 release

### DISCUSSION THREAD

**Discussion by Bry10022** [fontforge/fontforge#5516] [General]
Since it's almost been two years since the last release (in 2023), I wonder if we can expect a 2025 release to arrive anytime soon (The Unicode data used by FontForge should probably be updated to the latest version)?

**Comment by ultrasquid** [comment]
I agree. I would compile from source on my own but I don't know how to do that.

**Comment by iorsh** [comment]
Sounds like a good idea. I think we can have a release once the following items are closed:

- [x] #5336 - general performance and appearance configuration on Windows
- [x] #5551 - CI fix, **please merge it first**
- [x] #5496
- [x] #5522
- [x] #5537
- [x] #5546
- [x] #5547
- [x] fontforge/fontforgebuilds#4
- [x] jtanx/fontforgebuilds#25
- [x] #5575, update latest Spanish translations - see #5593
- [x] Update Metrics View documentation for HarfBuzz shaper: 007b8394fc78ed362e207504418f64c51321f6e2
- [x] Update latest Unicode data - see #5594 
- [ ] Update release id (currently 20230101) in the source
- [ ] Once released, update download links at https://fontforge.org
- [ ] Anything else?

https://github.com/fontforge/fontforge/wiki/Making-a-release
@skef @jtanx @davelab6 @JoesCat

**Reply by vasilky3** [reply]
I am also support idea of 2025 release. 
I would prefer to share https://fontforge.org/en-US/downloads/ page. Not the builded *.zip.

As i can see part of issues are features not bugfixes. So it can create new bugs. 
What is the purpose of waiting? Is release so hard to pull off (time-consuming)?
Just asking as an alternative opinion.

**Reply by jayaddison** [reply]
> * [ ] Update release id (currently 20230101) in the source

NB: I doubt that you would forget or not check, but just in case: please note that this version number exists not only in the build script (`CMakeLists.txt`), but also in the [documentation configuration file](https://github.com/fontforge/fontforge/blob/4c7c0579c8a16c95356069b79d28c8df60d2b450/doc/sphinx/conf.py#L19-L20) and [AppData XML](https://github.com/fontforge/fontforge/blob/4c7c0579c8a16c95356069b79d28c8df60d2b450/desktop/org.fontforge.FontForge.appdata.xml.in#L35).

**Comment by iorsh** [comment]
Fixed appearance configuration on Windows - see jtanx/fontforgebuilds#25

**Comment by iorsh** [comment]
Added #5551 to the list (CI fix)

**Comment by ultrasquid** [comment]
Issue #5209 (Spiro generating straight segments that should be curved) was
closed, but please check that it was actually resolved before releasing the
new build. I've had trouble with an interim solution.

On Sun, Apr 20, 2025 at 1:53 PM Maxim Iorsh ***@***.***>
wrote:

> Added #5551 <https://github.com/fontforge/fontforge/pull/5551> to the
> list (CI fix)
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/fontforge/fontforge/discussions/5516#discussioncomment-12892660>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AANUJCN3TZHWMVBCNUDPRIT22QCMNAVCNFSM6AAAAABUGLKTSKVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTEOBZGI3DMMA>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
-- 
Jason Pagura
zimbach at gmail dot com

**Reply by iorsh** [reply]
The most recent AppImage works. Verified the inlcuded libspiro version, looks ok.
`2025-04-21T07:25:03.2395675Z -- Found Libspiro: /home/runner/work/fontforge/fontforge/deps/install/lib/libspiro.so (found version "20240903")`

**Comment by ultrasquid** [comment]
Thank you very much. Sorry for being a bother.

On Mon, Apr 21, 2025 at 1:39 AM Maxim Iorsh ***@***.***>
wrote:

> The most recent AppImage works. Verified the inlcuded libspiro version,
> looks ok.
> 2025-04-21T07:25:03.2395675Z -- Found Libspiro:
> /home/runner/work/fontforge/fontforge/deps/install/lib/libspiro.so (found
> version "20240903")
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/fontforge/fontforge/discussions/5516#discussioncomment-12896790>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AANUJCLWEDXZDERFVRK2YBL22SVD5AVCNFSM6AAAAABUGLKTSKVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTEOBZGY3TSMA>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
-- 
Jason Pagura
zimbach at gmail dot com

**Comment by ultrasquid** [comment]
I agree with you.
When George Williams was running the project, he released a new build of
FontForge nearly every month whether it needed it or not. I wouldn't
necessarily request similar frequency, but more often than once a year
should certainly be doable.

On Mon, May 26, 2025 at 3:51 AM Vasilii ***@***.***> wrote:

> I am also support idea of 2025 release.
> I would prefer to share https://fontforge.org/en-US/downloads/ page. Not
> the builded *.zip.
>
> As i can see part of issues are features not bugfixes. So it can create
> new bugs.
> What is the purpose of waiting? Is release so hard to pull off
> (time-consuming)?
> Just asking as an alternative opinion.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/fontforge/fontforge/discussions/5516#discussioncomment-13271309>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AANUJCJWG23USBHWMTDFIID3ALWZNAVCNFSM6AAAAABUGLKTSKVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTGMRXGEZTAOI>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
-- 
Jason Pagura
zimbach at gmail dot com

**Comment by iorsh** [comment]
@brigazvi

**Comment by iorsh** [comment]
The biggest issue as of now is the localization strings (see discussion in #5575). I'm working on cleaning excessive newlines from translatable strings, to prevent future similar issues with translations.

I'm going on vacation until the beginning of August, after that I would be free to make a release.

@skef @frank-trampe @JoesCat

**Comment by iorsh** [comment]
October 2025 release PR #5632
